icons/swiprolog.svg

```diff
@@ -0,0 +1 @@
0 1 4.2517 4.2517 4.2516 4.2516 0 0 1-.065.5773 2.4152 2.4152 0 0 1 .686-.0995c.2443 0 .48.036.7026.103a4.2516 4.2516 0 0 1-.0656-.5807 4.2516 4.2516 0 0 1 4.2515-4.2518 4.2516 4.2516 0 0 1 4.2514 4.2518 4.2516 4.2516 0 0 1-4.2514 4.2514 4.2516 4.2516 0 0 1-2.5583-.8696c-.403 1.5642-2.33 3.4866-2.33 4.8984 0-1.4092-1.9199-3.327-2.3277-4.8897a4.2516 4.2516 0 0 1-2.5452.861A4.2516 4.2516 0 0 1 2.796 8.4587 4.2516 4.2516 0 0 1 7.0476 4.207zm.892 2.5956a2.066 2.066 0 0 0-2.0701 2.0701 2.066 2.066 0 0 0 2.07 2.07 2.066 2.066 0 0 0 2.07-2.07 2.066 2.066 0 0 0-2.07-2.07zm9.745 0a2.066 2.066 0 0 0-2.0701 2.0701 2.066 2.066 0 0 0 2.07 2.07 2.066 2.066 0 0 0 2.0701-2.07 2.066 2.066 0 0 0-2.07-2.07zm-9.745.7483A1.3216 1.3216 0 0 1 9.261 8.8727a1.3216 1.3216 0 0 1-1.3215 1.3218 1.3216 1.3216 0 0 1-1.3217-1.3218 1.3216 1.3216 0 0 1 1.3217-1.3218zm9.745 0a1.3216 1.3216 0 0 1 1.3217 1.3218 1.3216 1.3216 0 0 1-1.3218 1.3218 1.3216 1.3216 0 0 1-1.3217-1.3218 1.3216 1.3216 0 0 1 1.3217-1.3218zm-10.3344.4619a.398.398 0 0 0-.3981.398.398.398 0 0 0 .398.3981.398.398 0 0 0 .3982-.398.398.398 0 0 0-.3981-.3981zm9.7612 0a.398.398 0 0 0-.3981.398.398.398 0 0 0 .398.3981.398.398 0 0 0 .3982-.398.398.398 0 0 0-.3981-.3981zm-5.1972 1.323c-1.3217 0-1.9217.6946-1.9217 2.0004 0 1.64 1.5526 2.5802 1.9014 4.2352.4764-1.6629 2.0082-2.6548 2.0082-4.295 0-1.3216-.6663-1.9406-1.988-1.9406zM2 12.9173c-.1434.7484.6369 1.3215.6369 1.3215s.9396-.2229 1.083-.9713c-.1752.9395-1.2264 1.6402-1.2264 1.6402S1.809 13.841 2 12.9173zm19.9201 0c.1911.9236-.4935 1.9904-.4935 1.9904s-1.0509-.7007-1.226-1.6402c.1433.7484 1.0826.9713 1.0826.9713s.7802-.573.637-1.3215zm-18.0535.4519c-.099.714.6332 1.2518.6332 1.2518s.8773-.2082 1.0655-.8979c-.2278.8878-1.1927 1.5669-1.1927 1.5669s-.6504-1.0141-.506-1.9208zm16.2027 0c.1444.9067-.5057 1.9208-.5057 1.9208s-1.0352-.7167-1.2104-1.6403c.1433.7484 1.083.9713 1.083.9713s.7321-.5378.6331-1.2518zm-14.3715.366c-.099.714.6332 1.2522.6332 1.2522s.9396-.2229 1.083-.9713c-.1752.9235-1.2102 1.64-1.2102 1.64s-.6506-1.0141-.506-1.9208zm12.5245 0c.1445.9068-.506 1.9208-.506 1.9208s-.9659-.6794-1.193-1.5678c.1876.6904 1.0658.8992 1.0658.8992s.7322-.5381.6332-1.2521zM2.5414 14.9716c-.1433.7484.637 1.3215.637 1.3215s.9392-.223 1.0826-.9713c-.1752.9395-1.226 1.6402-1.226 1.6402s-.6847-1.0668-.4936-1.9904zm18.8373 0c.191.9236-.4936 1.9904-.4936 1.9904s-1.0508-.7007-1.226-1.6402c.1433.7484 1.0826.9713 1.0826.9713s.7803-.5731.637-1.3215zm-16.9744.382c-.1433.7483.637 1.3217.637 1.3217s.8745-.208 1.0645-.895c-.2299.8863-1.2079 1.5636-1.2079 1.5636s-.6847-1.0668-.4936-1.9904zm15.1114 0c.1911.9235-.4935 1.9903-.4935 1.9903s-1.035-.7163-1.2101-1.6399c.1433.7484 1.0826.9713 1.0826.9713s.7803-.5734.621-1.3218zm-16.4011 1.704c-.1592.7483.621 1.3214.621 1.3214s.8747-.2077 1.0646-.8948c-.229.8864-1.192 1.5637-1.192 1.5637s-.6847-1.0668-.4936-1.9904zm17.7033.0696c.1444.9067-.506 1.9208-.506 1.9208s-.963-.6774-1.192-1.5638c.19.6872 1.0645.8948 1.0645.8948s.7325-.5378.6335-1.2518z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>SWI Prolog</title><path d="M7.96 6.793a2.072 2.072 0 1 0 0 4.143 2.072 2.072 0 0 0 0-4.143zm0 3.397a1.326 1.326 0 1 1-.002-2.65 1.326 1.326 0 0 1 .001 2.65zm-.19-1.792a.397.397 0 1 1-.795 0 .397.397 0 0 1 .795 0zm9.953 2.538a2.072 2.072 0 1 0 0-4.143 2.072 2.072 0 0 0 0 4.143zm0-3.397a1.325 1.325 0 1 1 0 2.65 1.325 1.325 0 0 1 0-2.65zm-.984.86a.397.397 0 1 1 .794 0 .397.397 0 0 1-.794 0zm2.981-4.021c-.326-1.047-.718-3.103.705-4.296 0 0-2.746-.648-4.278 1.707A10.178 10.178 0 0 0 12 .907c-1.481 0-2.894.319-4.18.896C6.29-.57 3.532.082 3.532.082c1.457 1.221 1.013 3.345.683 4.366C2.381 6.49 1.253 9.25 1.253 12.29c0 2.42 1.554 5.169 1.865 8.853l1.968-1.705s.547.367 1.612.748A2.482 2.482 0 0 0 6.403 24a1.995 1.995 0 0 1-.01-.199c0-1.03.807-1.865 1.802-1.865.996 0 1.803.835 1.803 1.865l-.002.059a2.474 2.474 0 0 0 .276-2.909 18.31 18.31 0 0 0 3.255.019 2.472 2.472 0 0 0 .288 2.89c-.001-.02-.003-.039-.003-.059 0-1.03.807-1.865 1.803-1.865.996 0 1.803.835 1.803 1.865 0 .067-.004.133-.01.199a2.478 2.478 0 0 0-.234-3.767c1.153-.398 1.74-.795 1.74-.795l1.968 1.705c.312-3.684 1.865-6.434 1.865-8.853 0-3.075-1.155-5.864-3.027-7.912zm-17.683 8.55c-.147.744.63 1.316.63 1.316s.934-.233 1.081-.976c-.184.928-1.217 1.657-1.217 1.657s-.677-1.067-.494-1.996zm.543 2.055c-.147.743.63 1.315.63 1.315s.933-.233 1.08-.976c-.183.929-1.217 1.657-1.217 1.657s-.677-1.067-.493-1.996zm1.937-.367s.934-.233 1.08-.976c-.183.928-1.216 1.657-1.216 1.657s-.677-1.067-.494-1.996c-.147.743.63 1.315.63 1.315zm-.894 4.446s-.677-1.067-.493-1.996c-.147.744.63 1.316.63 1.316s.933-.234 1.08-.976c-.183.928-1.217 1.656-1.217 1.656zm1.3-1.711s-.677-1.067-.493-1.996c-.147.743.629 1.315.629 1.315s.934-.233 1.081-.976c-.184.929-1.217 1.657-1.217 1.657zm1.3-1.684s-.676-1.067-.493-1.996c-.147.744.63 1.316.63 1.316s.934-.234 1.081-.976c-.184.928-1.217 1.656-1.217 1.656zm5.717 1.07c0-1.403-1.908-3.327-2.319-4.891a4.222 4.222 0 0 1-2.542.85 4.247 4.247 0 1 1 4.247-4.246c0 .19-.017.376-.042.56a2.395 2.395 0 0 1 1.354.01 4.263 4.263 0 0 1-.043-.57 4.247 4.247 0 1 1 4.247 4.247c-.97 0-1.862-.33-2.578-.878-.395 1.57-2.324 3.507-2.324 4.918zm8.152-3.436c.184.929-.493 1.996-.493 1.996s-1.034-.729-1.217-1.657c.147.743 1.081.976 1.081.976s.776-.572.63-1.315zm-3.554.71c.148.742 1.082.976 1.082.976s.776-.572.63-1.316c.183.93-.494 1.996-.494 1.996s-1.034-.728-1.218-1.656zm1.302 1.683c.147.743 1.08.976 1.08.976s.777-.572.63-1.315c.184.929-.493 1.996-.493 1.996s-1.034-.728-1.217-1.657zm2.516 3.368s-1.033-.728-1.217-1.656c.147.742 1.082.976 1.082.976s.775-.572.629-1.316c.183.93-.494 1.996-.494 1.996zm.55-2.083s-1.033-.728-1.217-1.657c.147.743 1.081.976 1.081.976s.777-.572.63-1.315c.183.929-.494 1.996-.494 1.996zm-.675-3.711c.148.743 1.082.976 1.082.976s.776-.572.63-1.315c.183.929-.494 1.996-.494 1.996s-1.034-.729-1.218-1.657zm-6.244-1.97c0 .728-.592 1.674-1.166 2.589-.33.526-.65 1.038-.881 1.531-.232-.493-.552-1.005-.882-1.531-.573-.915-1.165-1.861-1.165-2.589A2.04 2.04 0 0 1 11.94 9.27a2.04 2.04 0 0 1 2.047 2.03z"/></svg>
```

Unfortunately I still see [some differences.](https://github.com/simple-icons/simple-icons/assets/35000807/72646c7e-eb81-42c3-aa10-b1e5625b3590)
 This suggestion should fix it but I will request someone else to review @simple-icons/icon-review